src/assets/scss/modules/advantages.scss

```diff
@@ -1,31 +0,0 @@
-.advantages {
-  @include backgroundImg("/assets/img/advateges.png");
-
-  &__items {
-    display: flex;
-  }
-
-  &__item {
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-    justify-content: flex-end;
-    flex: 1;
-    padding: 28px 0;
-    border-right: 1px solid #d4d4d4;
-  }
-
-  &__item:first-child {
-    border-left: 1px solid #d4d4d4;
-  }
-
-  &__text {
-    margin-top: 20px;
-    color: black;
```

Comment:
Try to follow the same code style through the whole project. Because now you set color using the keyword, somewhere else using rgb or hex code.